BatteryStatusDisplay.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace BatteryManagement
+{
+    public class BatteryStatusDisplay

```

Comment:
Why Print**** functions returning Boolean ? 
Why BatteryStatusDisplay Methods are non-static (None of these methods require any object state).
Separate Display functions from test functions

Reply:
Updated the files